packages/core/src/blocks/inputBlock.serializer.ts

```diff
@@ -43,7 +43,10 @@ function serializeInputBlockData(inputBlock: InputBlockBase): SerializedInputBlo
 function serializeTextureInputBlock(inputBlock: InputBlock<ConnectionPointType.Texture>): TextureInputBlockData {
     return {
         inputType: ConnectionPointType.Texture,
-        url: inputBlock.runtimeValue.value?.getInternalTexture()?.url || null,
+        url: inputBlock.runtimeValue.value?.getInternalTexture()?.url ?? null,
+        flipY: inputBlock.editorData?.flipY ?? null,
```

Comment:
I wonder if we should use the info from the texture instead so if we create by code, it works greats to ?

Reply:
Good suggestion, done!